spec/unit/project_spec.rb

```diff
@@ -235,7 +235,7 @@ def settings_for_root_configs(key)
           end.message.should.match /Paths must be absolute/
         end
 
-        it 'uses realpath for resolving symlinks' do
+        it 'retains original symlink paths' do
```

Comment:
Symlinks added to the project will now retain the symlink path rather than the real path